src/data/corpus.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+document = str(input("Please insert the full path to the text (without quotation marks): "))
```

Comment:
Next level: can you use command line arguments instead of inputs?
Hint: check argparser library